### ISSUE TITLE
Missing config dependency

### ISSUE THREAD

**Issue by LordEnder-Kitty** [Octo-Studios/immersive-ui#7]
The mod requires a another mod for configuration, yet this is not listen in the fabric.mod.json, causing the game to crash without it but without making it obvious what is missing or what went wrong.

**Comment by skellitor301** [comment]
Yeah, this needs Octolib which the mod page lists as required. FML doesn't list the dependency, just the error mentioned in this issue #10. It'd help if this was listed as a required library on the mod page.
